infra/packer/Dockerfile.build

```diff
@@ -32,13 +31,15 @@ RUN mv build/umeed build/umeed-beta
 # Build umeed
 RUN make build
 
-# Build price-feeder
-WORKDIR /tmp/build/umee/price-feeder
+# Clone/Build price-feeder
+WORKDIR /tmp/build/price-feeder
+RUN git clone https://github.com/umee-network/umee.git --branch price-feeder/v0.1.1 --single-branch --depth 1 .
+WORKDIR /tmp/build/price-feeder/price-feeder
```

Comment:
Why is this change necessary?

Reply:
@alexanderbez previously price-feeder was just tracking the same git sha as umeed; since you implemented independent price-feeder versioning i'm taking advantage of that now.